stock_customer_deposits/readme/USAGE.md

```diff
@@ -0,0 +1,38 @@
+To use this module, you need to:
+
+Create Customer deposits
+
+1. Go to Sales > Quotations.
+2. Create new quotation to the desired customer
+3. Add storable product in order lines.
+4. Select route **Customer Deposit** in line.
+5. Add quantity you will keep as a deposit.
+6. Confirm quotation.
+7. Two pickings will be created.
+8. Validate first picking.
```

Comment:
**issue:** I see 2 pickings, but the 1st I see is the 2nd that was created, so this sentence becomes confusing.

Should I validate the one that appears first, or the one created first?

![image](https://github.com/OCA/stock-logistics-workflow/assets/973709/425b7df7-c719-4380-ba9f-186b4b03604e)

Please clarify in docs.